models/_sector/nft/approvals/legacy/nft_arbitrum_approvals_legacy.sql

```diff
@@ -1,6 +1,6 @@
 {{ config(
-	tags=['legacy'],
-	
+	tags=['legacy', 'remove'],
```

Comment:
What's the `remove` tag here?

Reply:
> What's the `remove` tag here?
This tag will disable the legacy model (both on dbt cloud as in the CI)
It was introduced here: https://github.com/duneanalytics/spellbook/pull/4188

Reply:
Ah this was introduced when I was off. Looks great!